modules/op-resolver-cluster.adoc

```diff
@@ -22,7 +22,7 @@ When creating a pipeline run, you can specify a pipeline that exists on the same
   - name: namespace
     value: <namespace>
   - name: kind
-    value: [pipeline|task]
+    value: [pipeline|task|step]
```

Comment:
@vdemeester shouldn't it be stepaction?

Reply:
yes, `stepaction` (see [here](https://github.com/tektoncd/pipeline/blob/main/docs/cluster-resolver.md#parameters))